src/sidebar/components/test/tag-editor-test.js

```diff
@@ -234,13 +238,28 @@ describe('TagEditor', function() {
       wrapper.find('input').instance().value = 'tag3';
       selectOptionViaEnter(wrapper);
       assertAddTagsSuccess(wrapper, ['tag1', 'tag2', 'tag3']);
+      // ensure focus is still on the input field
+      assert.equal(document.activeElement.nodeName, 'INPUT');
     });
 
     it('adds a tag from the <input> field when typing "," delimiter', () => {
       const wrapper = createComponent();
       wrapper.find('input').instance().value = 'tag3';
       selectOptionViaDelimiter(wrapper);
       assertAddTagsSuccess(wrapper, ['tag1', 'tag2', 'tag3']);
+      // ensure focus is still on the input field
+      assert.equal(document.activeElement.nodeName, 'INPUT');
+    });
+
+    it('adds a tag when the <input> matches a suggestion and "Tab" is pressed', () => {
```

Comment:
Super pedantic, but perhaps this test description could be amended to:

`adds a tag when the <input> value is a match for a suggestion and "Tab" is pressed`

(i.e. it's the input's value that matters, not the input element overall)